tests/acceptance/user-views-projects-test.js

```diff
@@ -28,12 +29,15 @@ test("User views projects", function() {
     }
   });
 
-  visit("/projects").
-    then(function() {
-      equal(find(".project img:first").attr("src"), "cover.jpg");
-    }).
-  click(".next").
-    then(function() {
-      equal(find(".project iframe:first").attr("src"), "video.mp4");
-    });
+  visit("/projects");
+
+  andThen(() => {
+    assert.equal(find(".project img:first").attr("src"), "cover.jpg");
+  });
+
+  click(".next");
+
+  andThen(() => {
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>'andThen' is not defined.